src/main/java/organice/model/person/Donor.java

```diff
@@ -75,6 +78,10 @@ public Status getStatus() {
         return status;
     }
 
+    public ArrayList getPatientMatchedBefore() {
+        return patientsMatchedBefore;
```

Comment:
You may want to apply some form of defensive programming in here. I suggest the following code to make a new copy of the ArrayList before returning:

```
return new ArrayList<>(patientsMatchedBefore);
```

Reply:
done

Reply:
Noted, thank you.